src/inspector/view/qml/MuseScore/Inspector/text/TextInspectorView.qml

```diff
@@ -93,24 +93,26 @@ InspectorSectionView {
 
                     model: [
                         { iconCode: IconCode.TEXT_BOLD, value: TextTypes.FONT_STYLE_BOLD, title: qsTrc("inspector", "Bold") },
-                        { iconCode: IconCode.TEXT_ITALIC, value: TextTypes.FONT_STYLE_ITALIC, title: qsTrc("inspector", "Italic") },
+                        { iconCode: IconCode.TEXT_ITALIC, value: TextTypes.FONT_STYLE_ITALIC, title: qsTrc("inspector", "Italics") },
                         { iconCode: IconCode.TEXT_UNDERLINE, value: TextTypes.FONT_STYLE_UNDERLINE, title: qsTrc("inspector", "Underline") },
-                        { iconCode: IconCode.TEXT_STRIKE, value: TextTypes.FONT_STYLE_STRIKE, title: qsTrc("inspector", "Strike-through") }
+                        { iconCode: IconCode.TEXT_STRIKE, value: TextTypes.FONT_STYLE_STRIKE, title: qsTrc("inspector", "Strikethrough") }
                     ]
 
                     delegate: FlatToggleButton {
                         navigation.panel: root.navigationPanel
                         navigation.name: "FontStyle" + model.index
                         navigation.row: styleGroup.navigationRowStart + model.index
-                        navigation.accessible.name: styleSection.titleText + " " + modelData["title"]
+                        navigation.accessible.name: modelData.title
 
-                        icon: modelData["iconCode"]
+                        toolTipTitle: modelData.title
 
-                        checked: root.model && !root.model.fontStyle.isUndefined ? root.model.fontStyle.value & modelData["value"] : false
+                        icon: modelData.iconCode
+
+                        checked: root.model && !root.model.fontStyle.isUndefined ? root.model.fontStyle.value & modelData.value : false
```

Comment:
If you're going to change this line, this would be much clearer:
```QML
checked: (root.model && !root.model.fontStyle.isUndefined)
            ? (root.model.fontStyle.value & modelData.value)
            : false
```

Or even this:
```QML
checked: root.model && !root.model.fontStyle.isUndefined && (root.model.fontStyle.value & modelData.value)
```

I've seen the `expression1 ? expression2 : false` pattern in a few places. I don't know why we don't just do `expression1 && expression2`. If `expression1` is false then `expression2` will not be evaluated in either case.